cli/src/athenapdf.js

```diff
@@ -191,6 +191,15 @@ app.on("ready", () => {
         }
     });
 
+    bw.webContents.on("did-get-response-details", (e, status, newURL, originalURL, httpResponseCode, requestMethod, referrer, headers, resourceType) => {
+        if (httpResponseCode >= 400) {
+            console.error(`Failed to load ${newURL} - got HTTP code ${httpResponseCode}`);
+            if (originalURL === uriArg) {
```

Comment:
I believe what you want here is better achieved with checking if `resourceType` is `mainFrame`.